walter_nicholas/package.json

```diff
@@ -14,8 +14,21 @@
     "mongoose": "^4.2.5"
   },
   "devDependencies": {
+    "angular": "^1.4.8",
+    "angular-mocks": "^1.4.8",
+    "bcrypt": "^0.8.5",
```

Comment:
many of these are not a devDependency but are dependencies. Eat, bcrypt, body-parser.